iothub/device/readme.md

```diff
@@ -15,8 +15,6 @@ The library is available as a NuGet package for you include in your own developm
 ## Application development guidelines
```

Comment:
Same: please remove - application development does not require setting up a dev environment to work on SDKs. Instead NuGet packages can be used.

Reply:
updated